### PR TITLE
docs: add code-of-conduct and contributing documents

### PR DESCRIPTION
**Pull request by Turupawn**
### Purpose or design rationale of this PR

Added the following documents that should be helpful when we open source this repo:
* Code of conduct: based on Covenant COD, recommended by github
* Contribute: guide for developers wanting to contribute to this repo based on [this template](https://github.com/probot/template/blob/master/CONTRIBUTING.md)

### PR title

Your PR title must follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (as we are doing squash merge for each PR), so it must start with one of the following [types](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type):

- [ ] build: Changes that affect the build system or external dependencies (example scopes: yarn, eslint, typescript)
- [ ] ci: Changes to our CI configuration files and scripts (example scopes: vercel, github, cypress)
- [x] docs: Documentation-only changes
- [ ] feat: A new feature
- [ ] fix: A bug fix
- [ ] perf: A code change that improves performance
- [ ] refactor: A code change that doesn't fix a bug, or add a feature, or improves performance
- [ ] style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- [ ] test: Adding missing tests or correcting existing tests

### Deployment tag versioning

Has `tag` in `common/version.go` been updated?

- [x] No, this PR doesn't involve a new deployment, git tag, docker image tag
- [ ] Yes

### Breaking change label

Does this PR have the `breaking-change` label?

- [x] No, this PR is not a breaking change
- [ ] Yes

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+TODO.
```
Comment (Thegaram) [inline CODE_OF_CONDUCT.md]: Asking @dghelm what email we should use here.

**Review by Thegaram — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -90,6 +90,10 @@ You can find the unit tests in [`contracts/src/test/`](/contracts/src/test/), an
 
 See [`contracts`](/contracts) for more details on the contracts.
 
+## Contributing
+
+We welcome community contributions to this repository. Before you submit any issues or PRs, please read the [Code of Conduct](CODE_OF_CONDUCT.md) and [CONTRIBUTING.md](CONTRIBUTING.md).
```
Comment (icemelon) [inline README.md]: This should look better imo.

```suggestion
We welcome community contributions to this repository. Before you submit any issues or PRs, please read the [Code of Conduct](CODE_OF_CONDUCT.md) and [Contribution Guideline](CONTRIBUTING.md).
```

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (icemelon) [reply CODE_OF_CONDUCT.md]: how about contributor@scroll.io ? I can create a mailing list

File: README.md [hunk 3]
```diff
@@ -90,6 +90,10 @@ You can find the unit tests in [`contracts/src/test/`](/contracts/src/test/), an
 
 See [`contracts`](/contracts) for more details on the contracts.
 
+## Contributing
```
Comment (icemelon) [inline README.md]: We can move this section before `Prerequisites`

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Thegaram) [reply CODE_OF_CONDUCT.md]: Sure, sounds good

Comment (icemelon) [reply CODE_OF_CONDUCT.md]: created

**Review by icemelon — DISMISSED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (icemelon) [reply CODE_OF_CONDUCT.md]: added to the doc

**Review by Thegaram — APPROVED** [review]

**Review by icemelon — APPROVED** [review]

**Pull request merged**